src/hooks/useBackButton.tsx

```diff
@@ -19,8 +19,8 @@ export default function useBackButton({ username }: Props) {
   const backDisabled = previousRoute?.includes('/edit?collectionId=');
 
   const handleBackClick = useCallback(
-    (event: React.MouseEvent<HTMLElement>) => {
-      if (event.metaKey) {
+    (event?: React.MouseEvent<HTMLElement>) => {
+      if (event?.metaKey) {
```

Comment:
This enables the use of `useBackButton` without providing an event (it is traditionally triggered via click but this frees us up to use the same functionality elsewhere)